osCam/userconfig/views.py

```diff
@@ -1,111 +1,231 @@
-from ipaddress import ip_address
-from django.http import JsonResponse
-from django.shortcuts import render
-import json as Json
-
-# from core import models
-from userconfig.models import Network, Camera, RaspberryPi
 from django.contrib.auth.models import User
-# from userconfig.models import Network
-
-################################
-################################
-#  Views used for User Configurations: .
-#       e.g. A Config is of Type::{RaspberryPi, Camera, Network}
-def configPage(request):
-    '''
-        View Handles Http response To Display all user defined Configuration.
-            Configuration Could be of Type::{RaspberryPi, Camera, Network}.
-        
-        @param  request: Request 
-    '''
-    page = "User Configuration Page"
+from django.shortcuts import render, redirect
+from userconfig.forms import *
+from userconfig.models import *
+
+def settings(request):
```

Comment:
To meet Complexity goals, we should take each large conditional block and put it in it's own function which would then just be called from the settings function.